torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3689,6 +3689,14 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypes=all_types_and_complex_and(torch.float16, torch.bfloat16, torch.bool),
            supports_out=False,
            sample_inputs_func=sample_inputs_reduction_wrapper(supports_multiple_dims=True)),
+    # TODO Add test for dtype kwarg
```

Comment:
are you planning to work on it later? If yes, could you add your github id along with the TODO --> `TODO(@heitorschueroff): `

Reply:
Yes, I'll address it when I implement a ReductionInfo subclass of OpInfo and port these tests to use it.